superset/migrations/versions/3ba29ecbaac5_change_datatype_of_type_in_basecolumn.py

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Change datatype of type in BaseColumn
+
+Revision ID: 3ba29ecbaac5
+Revises: b92d69a6643c
+Create Date: 2021-11-02 17:44:51.792138
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "3ba29ecbaac5"
+down_revision = "b92d69a6643c"
```

Comment:
I think your tests are failing because your migration is now introducing a second HEAD. You can change the down revision here and in line 20 to `abe27eaf93db` (which you can see if you run `superset db heads`).